persistent-mongoDB/Database/Persist/MongoDB.hs

```diff
@@ -430,15 +445,18 @@ toInsertDoc record =
                 DB.:=
                     embeddedVal pv
         )
-        $ filter (\(_, pv) -> isNull pv)
+        $ filter (\(_, pv) -> not $ isNull pv)
```

Comment:
The major fix was this! At least for our needs this was the only thing that really broke anything.

Reply:
Wow. I'm really surprised that worked?